diff --git a/node_modules/metro-config/src/defaults/getMaxWorkers.js b/node_modules/metro-config/src/defaults/getMaxWorkers.js
index 1a23754..d2faa73 100644
--- a/node_modules/metro-config/src/defaults/getMaxWorkers.js
+++ b/node_modules/metro-config/src/defaults/getMaxWorkers.js
@@ -9,7 +9,7 @@ function _interopRequireDefault(e) {
   return e && e.__esModule ? e : { default: e };
 }
 function getMaxWorkers(workers) {
-  const cores = _os.default.availableParallelism();
+  const cores = _os.default.availableParallelism ? _os.default.availableParallelism() : _os.default.cpus().length;
   return typeof workers === "number" && Number.isInteger(workers)
     ? Math.min(cores, workers > 0 ? workers : 1)
     : Math.max(1, Math.ceil(cores * (0.5 + 0.5 * Math.exp(-cores * 0.07)) - 1));
diff --git a/node_modules/metro-config/src/loadConfig.js b/node_modules/metro-config/src/loadConfig.js
index 7ac9d88..fe3c7ba 100644
--- a/node_modules/metro-config/src/loadConfig.js
+++ b/node_modules/metro-config/src/loadConfig.js
@@ -176,12 +176,12 @@ async function mergeConfigAsync(baseConfig, ...reversedConfigs) {
 }
 function mergeConfig(base, ...configs) {
   let currentConfig = typeof base === "function" ? base() : base;
-  const reversedConfigs = configs.toReversed();
+  const reversedConfigs = configs.slice().reverse();
   let next;
   while ((next = reversedConfigs.pop())) {
     const nextConfig = typeof next === "function" ? next(currentConfig) : next;
     if (nextConfig instanceof Promise) {
-      return mergeConfigAsync(nextConfig, reversedConfigs.toReversed());
+      return mergeConfigAsync(nextConfig, reversedConfigs.slice().reverse());
     }
     currentConfig = mergeConfigObjects(currentConfig, nextConfig);
   }
